modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -77,27 +77,29 @@ The `--copy-network` option only copies networking configuration found under `/e
+* Creating separate partitions: For greenfield installations on an empty disk, you might want to add separate storage to a partition. For example, mount `/var/lib/containers` in a separate partition, so that kubelet can monitor the `imagefs` directory for container runtimes,`/var/lib/containers` and the `nodefs` directory for root directory files.
 +
 [IMPORTANT]
 ====
-For disk sizes larger than 100GB, and especially disk sizes larger than 1TB, create a separate `/var` partition. See "Creating a separate `/var` partition" and this link:https://access.redhat.com/solutions/5587281[Red Hat Knowledgebase article] for more information.
+If you resized your disk size to host a larger file system, consider creating a separate `/var/lib/containers` partition. This is important for a disk formatted to `xfs`, where a high number of allocation groups might cause CPU time issues.
 ====
+// https://linux.die.net/man/5/xfs
+// https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/
+
+* Retaining existing partitions: For a brownfield installation where you are reinstalling {product-title} on an existing node and want to retain data partitions installed from your previous operating system, use boot arguments and options with the `coreos-installer` command so that you can keep existing data partitions.
 +
 [IMPORTANT]
 ====
-Kubernetes supports only two file system partitions. If you add more than one partition to the original configuration, Kubernetes cannot monitor all of them.
+The use of a custom partition for mounting ` /var/lib/containers` could result in the partition from not being monitored by {product-title} or alerted on. If you are overriding the default partitioning, see link:https://access.redhat.com/articles/4766521[Understanding OpenShift File System Monitoring (eviction conditions)] for more information about how {product-title} monitors your host file systems.
 ====
 
-* Retaining existing partitions: For a brownfield installation where you are reinstalling {product-title} on an existing node and want to retain data partitions installed from your previous operating system, there are both boot arguments and options to `coreos-installer` that allow you to retain existing data partitions.
-
-[WARNING]
+[IMPORTANT]
 ====
-The use of custom partitions could result in those partitions not being monitored by {product-title} or alerted on. If you are overriding the default partitioning, see link:https://access.redhat.com/articles/4766521[Understanding OpenShift File System Monitoring (eviction conditions)] for more information about how {product-title} monitors your host file systems.
+For disk sizes larger than 100GB, and especially disk sizes larger than 1TB, create a separate `/var` partition. See "Creating a separate `/var` partition" and this link:https://access.redhat.com/solutions/5587281[Red Hat Knowledgebase article] for more information.
```

Comment:
I don't really have the context for why this was in here. I'd leave it honestly.

Reply:
Thanks, I'll leave this note in place to be safe.

Reply:
To me, this very much look a legacy advice, where do you find a disk smaller than 100 GB in the past decade or so.  
If we can't trace the origin of the note through git history and we can't motivate why it should be this way, I suggest the note should be removed.

Reply:
it was added at the request of the CoreOS team, you can find more information on the upstream issue [here](https://github.com/coreos/fedora-coreos-tracker/issues/1183#issuecomment-1107857954). @control-d might be able to say whether this note is still needed or if it can be removed. my suggestion is to keep it, as this seemed to be such a common issue customers were running into

Reply:
reading the original [issue](https://github.com/coreos/fedora-coreos-tracker/issues/1183#issuecomment-1107857954), it refers to https://github.com/coreos/fedora-coreos-config/pull/2320 which supposedly solves the problem.  
And indeed, checking an OCP 4.14.5 Node I had, with a 828 GB root disk, the `agcount` was only 4. Which suggests that the problem for which this suggestion was created has been resolved

Reply:
Hi @djuran . Thanks for the comment, so the following note can be removed from only the 4.15 docs? 

> For disk sizes larger than 100GB, and especially disk sizes larger than 1TB, create a separate `/var` partition. See "Creating a separate `/var` partition" and this link:https://access.redhat.com/solutions/5587281[Red Hat Knowledgebase article] for more information.

Issue persists in earlier versions (4.14 to 4.11)?

Reply:
I'm not quite sure how to map CoreOS merges into OCP releases but at least 4.14.5 does *not* suffer from this issue, which suggests that note should be removed also for 4.14

Reply:
Thanks, @djuran . This would mean that I would have to make 4.12 and 4.13 updates in a separate PR. I'll target this PR for 4.14 and 4.15 so that I'll remove: 

> For disk sizes larger than 100GB, and especially disk sizes larger than 1TB, create a separate `/var` partition. See "Creating a separate `/var` partition" and this link:https://access.redhat.com/solutions/5587281[Red Hat Knowledgebase article] for more information.

Reply:
OCP 4.12:
```
[akaris@workstation k8s (master)]$ oc get clusterversion
NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.12.30   True        False         29d     Cluster version is 4.12.30
``

```
[root@30-d0-42-da-54-80 ~]# xfs_info / | grep agcount
meta-data=/dev/sda4              isize=512    agcount=502, agsize=233599 blks
[root@30-d0-42-da-54-80 ~]# df -h /
Filesystem      Size  Used Avail Use% Mounted on
/dev/sda4       447G  225G  223G  51% /
```

Reply:
Perhaps we should just target this for OCP 4.14, and I assume 4.13 as it's RHEL 9 as well, and ignore 4.12 for starters and have this discussion for the PR for that version - i.e., we can drop the sentence altogether for 4.14 (definitely) and 4.13 (very likely)

Reply:
Thanks, @andreaskaris . This would be possible but we need complete clarity that the note could be dropped  for 4.15, 4.14, and 4.13. Who can validate this? It seems that certain z-streams could still require the note, such as 4.14.6?